controllers/hook/SellermaniaImportOrder.php

```diff
@@ -255,7 +261,11 @@ public function preprocessData()
                 $this->data['OrderInfo']['Product'][$kp]['Amount']['Price'] = 0;
 
             // Get Product Identifiers
-            $this->data['OrderInfo']['Product'][$kp] = $this->getProductIdentifier($this->data['OrderInfo']['Product'][$kp]);
+            // Modif YB : data sent by reference
+			// the data has been sent by reference, it should not be returned, this is redundant
```

Comment:
I added this because on some users' PHP versions, reference was not used properly (variable was filled with wrong data). I even did some tests with dummy scripts and encounter the same issue. That's the reason why I send back data now.